stable/mongodb-replicaset/templates/mongodb-pod-services.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.podServices.enabled }}
+# Services to expose pods individually
+{{ $podCount := .Values.replicas | int }}
+{{- range $i := until $podCount }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
```

Comment:
This annotation should only be necessary for the governing headless service for the statefulset.